tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -890,7 +998,7 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
                         invocationsAllowed += 1
                         actionResult = Some(JsObject(
                             "headers" -> JsObject(
-                                "content-type" -> "xyz/bar".toJson),
+                                "content-type" -> "xyzbar".toJson),
```

Comment:
@akrabat, another way would be to create list of custom media types and content types for what you need.

Something like this for instance:
```
val jsonPatchMediaType = MediaType.applicationWithOpenCharset("application/json-patch+json") 
val jsonPatchContentType = jsonPatchMediaType withCharset HttpCharsets.`UTF-8` 
```